### DISCUSSION TITLE
How to generate random data for a class passed as string?

### DISCUSSION THREAD

**Discussion by edreux** [bchavez/Bogus#476] [Q&A]
I want to create a form where I can enter the definition of a c# class / model in a text field.
I want to generate random instances of it.
I have no idea how to proceed as all the samples I'm seeing will do mockup for a class belonging to the project and compiled with the project.
Any idea would be welcomed.

**Comment by ChristoWolf** [comment]
Hi @edreux!

Do you mean that you want to generate instances of classes with unknown type, e.g. anonymous types?
I would have a few ideas to do so.
